octobot_pro/__init__.py

```diff
@@ -1,4 +1,38 @@
+def _use_module_local_tentacles():
+    import sys
+    import os
```

Comment:
since we are here in the main entrypoint of the lib, I prefer not to expose too many things under the octobot_pro import. Importing those inside the local functions makes it so that `sys` is not visible in octobot_pro when importing it.
When in top file imports, this works as it is exposed by the module and i think it shouldn't:
```python
import octobot_pro as op
op.sys.[whatever sys func]()
```

Reply:
Ok thanks, it's clearer